app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -81,6 +106,38 @@ define(function (require, exports, module) {
       return ! account.has('profileImageUrl');
     },
 
+    _addLoadingSpinner: function (spinnerWrapperEl) {
+      return $('<span class="avatar-spinner"></span>').appendTo(spinnerWrapperEl.addClass('with-spinner'));
```

Comment:
It seems a bit janky to try to add content to a non-existent element, even if it doesn't cause any problems. I'm going to add the requested guard, just because it's good form.